features/user_sessions.feature

```diff
@@ -49,3 +49,21 @@ Feature: User sessions
     And the time becomes 2017-03-30 12:00
     And I follow "Add Site"
     Then I should be on the login page
+
+  @javascript
+  Scenario: Already logged-in super-user logs out
+    Given I am logged in with email "affiliate_admin@fixtures.org"
+    When I go to the usagov's Dashboard page
+    Then I should not see "Security Notification"
+    When I sign out
+    And I go to the usagov's Dashboard page
+    Then I should see "Security Notification"
```

Comment:
Changed
Also removed the 'When I sign out' line preceding commented out SRCH-1552 code, as it tests nothing useful after removing the commented out code (it was originally left in place because it wasn't clear whether would want it after this work).